.github/workflows/create-vercel-alias.yml

```diff
@@ -0,0 +1,32 @@
+
+name: 'Create Vercel Alias'
+
+on:
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: '22.x'
+
+jobs:
+  create-vercel-alias:
+    name: 'Create Vercel Alias'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Extract Current Version
+        id: extract_version
+        run: echo "VERSION=$(jq -r .version packages/components/package.json | sed 's/\./-/g')" >> $GITHUB_ENV
```

Comment:
Grabs the version from `package.json` and change format from dot-separator to dash-separator (e.g. `4.15.0` becomes `4-15-0`)